vllm/outputs.py

```diff
@@ -126,6 +128,18 @@ def from_seq_group(cls, seq_group: SequenceGroup,
                 not finished):
             return None
 
+        if finished and seq_group.request_id in group_id_to_holders:
+            # parallel sampling can not stream the output
```

Comment:
Why is this the case? Theoretically for parallel sampling I think we can just concurrently stream multiple outputs back. We can interleave the output streams of different samples.

Reply:
fixed in [245920f](https://github.com/vllm-project/vllm/pull/9302/commits/245920f17ae10675d98a4e5fc92a0e326cb3fc93)